main/src/main/scala/sbt/internal/librarymanagement/DependencyResolver.scala

```diff
@@ -0,0 +1,117 @@
+package sbt.internal.librarymanagement
+
+import java.io.File
+
+import sbt.internal.util.Types._
+import sbt.internal.util.HNil
+import sbt.librarymanagement._
+import sbt.util.{ CacheStore, CacheStoreFactory, Logger, Tracked }
+import sbt.librarymanagement.syntax._
+import sbt.util.CacheImplicits._
+
+object DependencyResolver {
```

Comment:
Same with above. This object should be called `LibraryManagementImpl` or something like that since `Resolver` has specific meaning.